src/op_handlers/context.rs

```diff
@@ -77,7 +77,8 @@ pub fn set_context_u128(vm: &mut VMState, opcode: &Opcode) -> Result<(), EraVmEr
 }
 
 pub fn aux_mutating0(_vm: &mut VMState, _opcode: &Opcode) -> Result<(), EraVmError> {
-    panic!("Not yet implemented");
+    // does nothing for now
+    Ok(())
```

Comment:
Disregard my last comment, calling this function should return an `Err(...)` much like an invalid opcode on lib.rs does, you can put the error here or on the matching branch for this instruction over on lib.rs

Reply:
That, wouldn't be compatible with their current behaviour. Both of their implementation just ignore the opcode and increment the pc. Returning err would trigger a ret.panic. Currently, none of the contracts we have tested falls here, so maybe a panic would be ok to spot it later, but in theory this opcode should just increment the pc.

Reply:
I think the best idea would be to return an `ExecutionOutput::Panic` disregarding any type of frame pop or anything, just end execution, if we just leave it with an ok, and someday someone adds a test that uses it it will be really hard to catch, but if we returned a Panic it would be easier to see that the last opcode executed was this one